xds/src/main/java/io/grpc/xds/orca/OrcaMetricReportingServerInterceptor.java

```diff
@@ -45,22 +48,31 @@
 public final class OrcaMetricReportingServerInterceptor implements ServerInterceptor {
 
   private static final OrcaMetricReportingServerInterceptor INSTANCE =
-      new OrcaMetricReportingServerInterceptor();
+      new OrcaMetricReportingServerInterceptor(null);
 
   @VisibleForTesting
   static final Metadata.Key<OrcaLoadReport> ORCA_ENDPOINT_LOAD_METRICS_KEY =
       Metadata.Key.of(
           "endpoint-load-metrics-bin",
           ProtoUtils.metadataMarshaller(OrcaLoadReport.getDefaultInstance()));
 
+  @Nullable
+  private final MetricRecorder metricRecorder;
+
   @VisibleForTesting
-  OrcaMetricReportingServerInterceptor() {
+  OrcaMetricReportingServerInterceptor(@Nullable MetricRecorder metricRecorder) {
+    this.metricRecorder = metricRecorder;
   }
 
-  public static OrcaMetricReportingServerInterceptor getInstance() {
+  public static OrcaMetricReportingServerInterceptor create() {
```

Comment:
Don't break the existing API. Even if we wanted to change the name, we'd leave the old name available and mark it `@Deprecated`.

I'm fine with changing the name, but I think it is @YifeiZhuang's call. I'd be tempted to make it a constructor, though.

Reply:
API review meeting notes:
 * Keep the current `getInstance()`, behaving the same as it was previously
 * The new `create(MetricRecorder)` method looks fine

Reply:
Done. Updated name from `create()` back to `getInstance()`.